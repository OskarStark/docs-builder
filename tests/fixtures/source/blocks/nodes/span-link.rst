`README file with version substitution <https://github.com/symfony/amazon-sns-notifier/blob/{version}/README.md>`_

`README file without version substitution <https://github.com/symfony/amazon-sns-notifier/blob/version/README.md>`_