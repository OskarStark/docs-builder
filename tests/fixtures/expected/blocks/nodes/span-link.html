<p><a href="https://github.com/symfony/amazon-sns-notifier/blob/4.0/README.md" class="reference external" rel="external noopener noreferrer" target="_blank">README file with version substitution</a></p>

<p><a href="https://github.com/symfony/amazon-sns-notifier/blob/version/README.md" class="reference external" rel="external noopener noreferrer" target="_blank">README file without version substitution</a></p>